prettytime.py

```diff
@@ -110,6 +110,10 @@ def __init__(self, num=None):
             self.num = None
             self.today = datetime.datetime.today()
 
+    def _print(self):
+        time_units = [attr for attr in dir(self) if attr in self.TIME_LIST and getattr(self, attr) != 0]
```

Comment:
use this logic in print instead:

if datetime.datetime.today() < self.today:
            return str(self.num) + " " + self.attr + " from now"
        else:
            return str(self.num) + " " + self.attr + " ago"